gpu-operator/dra-driver.rst

```diff
@@ -0,0 +1,691 @@
+     - Specifies the driver root on the host.
+       For GPU Operator-managed drivers (recommended), use ``/run/nvidia/driver``.
+       For pre-installed drivers, use ``/``.
+     - ``/``
+
+   * - ``nvidiaCtkPath``
+     - Specifies the path of the NVIDIA Container Toolkit CLI binary (nvidia-ctk) on the host.
+       For GPU Operator-installed NVIDIA Container Toolkit (recommended), use ``/usr/local/nvidia/toolkit/nvidia-ctk``.
+       For a pre-installed NVIDIA Container Toolkit, use ``/usr/bin/nvidia-ctk``.
+     - ``/usr/bin/nvidia-ctk``
+
+   * - ``resources.gpus.enabled``
+     - Specifies whether to enable the NVIDIA DRA Driver for GPUs to manage GPU resource allocation.
+       This feature is in Technolody Preview and only recommended for testing, not production enviroments.
+       To use with MNNVL, set to ``false``.
+     - ``true``
+
+
+Verify installation
+===================
+
+#. Validate that the components are running and in a ``READY`` state.
+
+   .. code-block:: console
+
+      $ kubectl get pod -n nvidia-dra-driver-gpu
+
+   *Example Output*
+
+   .. code-block:: output
+
+      NAME                                                           READY   STATUS    RESTARTS   AGE
+      nvidia-dra-driver-k8s-dra-driver-controller-67cb99d84b-5q7kj   1/1     Running   0          7m26s
+      nvidia-dra-driver-k8s-dra-driver-kubelet-plugin-7kdg9          1/1     Running   0          7m27s
+      nvidia-dra-driver-k8s-dra-driver-kubelet-plugin-bd6gn          1/1     Running   0          7m27s
+      nvidia-dra-driver-k8s-dra-driver-kubelet-plugin-bzm6p          1/1     Running   0          7m26s
+      nvidia-dra-driver-k8s-dra-driver-kubelet-plugin-xjm4p          1/1     Running   0          7m27s
+
+
+#. Confirm that all GPU nodes are labeled with clique ids.
+   The following command used `jq <https://jqlang.org/>`_ to format the output.
+
+   .. code-block:: console
+
+      $ (echo -e "NODE\tLABEL\tCLIQUE"; kubectl get nodes -o json | \
+        jq -r '.items[] | [.metadata.name, "nvidia.com/gpu.clique", .metadata.labels["nvidia.com/gpu.clique"]] | @tsv') | \
+        column -t
+
+   *Example Output*
+
+   .. code-block:: output
+
+      NODE                  LABEL                  CLIQUE
+      node1                 nvidia.com/gpu.clique  9277d399-0674-44a9-b64e-d85bb19ce2b0.32766
+      node2                 nvidia.com/gpu.clique  9277d399-0674-44a9-b64e-d85bb19ce2b0.32766
+
+The `NVIDIA GPU Feature Discovery <https://github.com/NVIDIA/k8s-device-plugin/tree/main/docs/gpu-feature-discovery>`_ adds a Clique ID to each GPU node.
+This is a unique identifier within an NVLink domain (physically connected GPUs over NVLink) that indicates which GPUs within that domain are physically capable of talking to each other.
+
+The partitioning of GPUs into a set of cliques is done at the NVSwitch layer, not at the individual node layer. All GPUs on a given node are guaranteed to have the same <ClusterUUID.Clique ID> pair.

```

Comment:
And why is it a pair? Oh, does the pair refer to the ClusterUUID and the Clique ID? Would "value" be better. I think of "pair" as something like (X,Y). Later on, "<cluster-uuid, clique-id>" is referenced, so this was confusing for me.

Within the bracket notation, would it be better if "Clique ID" were "Clique_ID" to make it clear that it is one value?